src/components/Search/Search.js

```diff
@@ -0,0 +1,21 @@
+import { TextField } from '@mui/material';
+
+import React from 'react';
+
+const Search = ({ setSearch, setPageNumber }) => {
```

Comment:
The `setPageNumber` doesn't belong here, this should be handled in the parent as an effect of `onChange`